cognite/neat/graph/stores/_base.py

```diff
@@ -124,11 +124,22 @@ def from_oxi_store(cls, storage_dir: Path | None = None, rules: InformationRules
         return cls(graph, rules)
 
     def write(self, extractor: TripleExtractors) -> None:
+        _start = datetime.now(pytz.utc)
```

Comment:
Maybe worth a discussion. Do we want have pytz as a dependency for neat? Even though it is useful, as neat is a python package, we should strive to have as few dependencies as possible.

Reply:
Agree, but it is not our direct dep. we can though remove it from usage.

Reply:
Fixed